pkg/config/config.go

```diff
@@ -7,36 +7,55 @@ import (
 	"strings"
 	"unicode"
 
-	"github.com/rs/zerolog/log"
+	"github.com/rs/zerolog"
 	flag "github.com/spf13/pflag"
 	"k8s.io/client-go/util/homedir"
 )
 
+type ZeroLogLevel int8
+
+func (l ZeroLogLevel) String() string {
+	return fmt.Sprintf("%d", l)
+}
+
+func (l ZeroLogLevel) Set(s string) error {
+	level, err := zerolog.ParseLevel(s)
+	if err != nil {
+		return err
+	}
+	l = ZeroLogLevel(int8(level))
```

Comment:
The double type conversion seems redundant, this works just fine:
```
        type ZeroLogLevel int8
        type SomeOther    int8

	var a SomeOther
	a = 10
	
	z := ZeroLogLevel(a)
	fmt.Printf("%d", z)
```
```suggestion
	l = ZeroLogLevel(level)
```